core/execution/future/market_snapshot.go

```diff
@@ -220,6 +220,12 @@ func NewMarketFromSnapshot(
 	marketType := mkt.MarketType()
 
 	markPriceCalculator := common.NewCompositePriceCalculatorFromSnapshot(ctx, em.CurrentMarkPrice, timeService, oracleEngine, em.MarkPriceCalculator)
+
+	allowedSellers := map[string]struct{}{}
```

Comment:
Looks like the `allowedSellers` variable is not used anywhere later.

Reply:
good point, it's a leftover of a previous version which used to implement the feature on both spots and futures. I'll remove it.